class StackOverflowError(Exception):
    pass


class StackUnderflowError(Exception):
    pass
